src/common/Notes/NoteCreate.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
is it really common? it's used only for POL, it looks like generic, but strictly connected to the route name

Reply:
I think purchase orders will have the same notes accordion